Gemfile.lock

```diff
@@ -121,36 +122,35 @@ GEM
       activesupport (>= 3.0.0)
       uniform_notifier (~> 1.11)
     byebug (11.1.3)
-    capistrano (3.14.0)
+    capistrano (3.14.1)
```

Comment:
### [capistrano](https://capistranorb.com/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.14.0 → 3.14.1

[_[change-log](https://github.com/capistrano/capistrano/releases), [source-code](https://github.com/capistrano/capistrano)_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/capistrano/capistrano/compare/v3.14.0...v3.14.1).

These are the individual commits:
- (800d840) [#2060 Fix ruby 2.7 keyword parameters warnings and bump up r…](https://github.com/capistrano/capistrano/commit/800d84065fcf2f5b12f0294c1b97e4d763a0c988)
- (e5600d7) [Release 3.14.1](https://github.com/capistrano/capistrano/commit/e5600d7c85a60d0cb77d33b5c6011e3ffb343660)


</details>